pontoon/base/static/js/fluent_interface.js

```diff
@@ -339,9 +343,18 @@ var Pontoon = (function (my) {
           }
           string += startMarker + '{' + element.expression.id.name + '}' + endMarker;
         }
-        else if (element.expression.type === 'CallExpression') {
+        else if (
+          [
+            'CallExpression',
+            'StringExpression',
+            'NumberExpression',
+            'VariantExpression',
+            'AttributeExpression',
+          ].indexOf(element.expression.type) >= 0
+        ) {
+          var title = element.expression.type.split('Expression')[0] + ' Expression';
```

Comment:
Interestingly, I would have gone for a `replace` here instead. 

```javascript
var title = element.expression.type.replace('Expression', ' Expression');
```

I think it's slightly more readable, as it expressed better what the code is doing. But it's a nit, so your call!

Reply:
True. I'll update.